### PR TITLE
Various fixes in Microsoft.NETCore.App packs

### PR DESCRIPTION
**Pull request by jkoritzinsky**
Include the fallback RID graph in the Microsoft.NETCore.App runtime pack.

Remove extraneous entries from the Microsoft.NETCore.App platform manifest and drive generation of the platform manifest entries for managed libraries from a list that already exists in the libraries portion of the repo.

### CODE REVIEW

**Comment by jkoritzinsky** [comment]
cc: @JunTaoLuo

**Review by sdmaclea — APPROVED** [review]
LGTM, but I don't fully understand it.

File: src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props [hunk 1]
```diff
@@ -62,7 +62,43 @@
...
   <ItemGroup>
+    <!-- Libraries specified in NetCoreAppLibrary.props -->
+    <PlatformManifestFileEntry Include="@(NetCoreAppLibrary->'%(Identity).dll')" />
+    <PlatformManifestFileEntry Include="@(NetCoreAppLibraryNoReference->'%(Identity).dll')" />
+    <!-- Shim libraries -->
+    <PlatformManifestFileEntry Include="mscorlib.dll" />
+    <PlatformManifestFileEntry Include="netstandard.dll" />
+    <PlatformManifestFileEntry Include="WindowsBase.dll" />
+    <PlatformManifestFileEntry Include="Microsoft.VisualBasic.dll" />
+    <PlatformManifestFileEntry Include="System.ComponentModel.DataAnnotations.dll" />
```
Comment (ericstj) [inline src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props:76]: You can actually drive these off https://github.com/dotnet/runtime/blob/master/src/libraries/shims/netfxreference.props

**Review by ericstj — APPROVED** [review]
This is OK to go as is.  Consider a follow up to reduce the list using netfxreference.props.

**Pull request merged**